portfolio/src/main/webapp/index.html

```diff
@@ -68,11 +76,80 @@ <h2 class="bios opener" id="computabeast"> AKA: THE computabeast </h2>
             </div>
         </div>
 
-        <div class="part">
+        <div class="part" id="secondpart">
+            <div id="links-table">
+                <table class="tabler">
+                        <td><img id="coolgif" src="images/5eKX.gif" alt="animated" /></td>
+                        <!-- https://gifer.com/en/5eKX -->
+                </table>
+            </div>
+
+             <div class="text-container" id="education">
```

Comment:
Now that you don't use the id's for searching, you can remove the excessive id's from this file.